tensorflow/python/compiler/tensorrt/test/trt_mode_test.py

```diff
@@ -157,8 +159,9 @@ def ExpectedEnginesToBuild(self, run_params):
 
   def ShouldRunTest(self, run_params):
     # Only run for TRT 6 and above.
-    return run_params.is_v2 and trt_test.IsTensorRTVersionGreaterEqual(6) and (
-        not run_params.use_calibration), "test v2 >=TRT6 and non-calibration"
+    return run_params.is_v2 and \
+        trt_utils.IsLinkedTensorRTVersionGreaterEqual(6) and \
```

Comment:
s/6/7/

Reply:
The removal of TRT 5 & 6 support will be done in a separate PR after this one.
I don't want to mix objectives.